File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/Delphi/IssueList/IssueListProjects.pas
unit IssueListProjects;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, IssueListForm, ImgList, ComCtrls, ToolWin, ExtCtrls, StdCtrls,
  cxControls, cxContainer, cxEdit, cxTextEdit, cxDBEdit, cxMaskEdit,
  cxDropDownEdit, cxLookupEdit, cxDBLookupComboBox, DBCtrls, cxGraphics,
  cxLookAndFeels, cxLookAndFeelPainters, cxDBLookupEdit, cxNavigator, cxLabel;

type
  TfrmProjects = class(TfrmBasic)
    cxDBTextEdit1: TcxDBTextEdit;
    Label5: TcxLabel;
    cxDBLookupComboBox3: TcxDBLookupComboBox;
    Label1: TcxLabel;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  frmProjects: TfrmProjects;

implementation

{$R *.dfm}

uses
  IssueListForms, IssueListConst, IssueListData;

initialization

  FormManager.RegisterForm(ProjectsFormID, TfrmProjects, 'PROJECTS');


end.

File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/CBuilder/IssueList/IssueListProjects.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IssueListProjects.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "IssueListForm"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxDBEdit"
#pragma link "cxDBLookupComboBox"
#pragma link "cxDBLookupComboBox"
#pragma link "cxDropDownEdit"
#pragma link "cxEdit"
#pragma link "cxLookupEdit"
#pragma link "cxMaskEdit"
#pragma link "cxTextEdit"
#pragma link "cxDBLookupComboBox"
#pragma link "cxDBLookupEdit"
#pragma link "cxGraphics"
#pragma link "cxLabel"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxLookAndFeels"
#pragma link "cxNavigator"
#pragma resource "*.dfm"
TfrmProjects *frmProjects;
//---------------------------------------------------------------------------
__fastcall TfrmProjects::TfrmProjects(TComponent* Owner)
  : TfrmBasic(Owner)
{
}
//---------------------------------------------------------------------------
